fix cb eedr testplaybook (#35542)
demisto/content: Packs/CarbonBlackEnterpriseEDR/ReleaseNotes/1_1_37.md

#### Integrations

##### Carbon Black Enterprise EDR

> - Fixed an issue in the **cb-list-alerts** command witch used to return an 500 error.

Reviewer: ```suggestion
Fixed an issue with the **cb-list-alerts*** command that used to return an 500 error.
```
Why did it return a 500 error?  What was the problem?

#### Integrations

##### Carbon Black Enterprise EDR

> - Fixed an issue in the ***cb-eedr-list-alerts*** command that failed when the user didn't specify *start* and *end* arguments.
> - Fixed an issue in the ***cb-eedr-process-search*** command that failed when specifying an *event_id* argument.

Reviewer 2: ```suggestion
- Fixed an issue where ***cb-eedr-list-alerts*** command failed if *start* and *end* arguments were not provided.
- Fixed an issue where ***cb-eedr-process-search*** command failed if *event_id* argument was specified.
```
